examples/experimental/dagster-airlift/dagster_airlift/constants.py

```diff
@@ -1,3 +1,3 @@
 MIGRATED_TAG = "airlift/task_migrated"
-DAG_ID_METADATA_KEY = "airlift/dag_id"
-TASK_ID_METADATA_KEY = "airlift/task_id"
+DAG_ID_METADATA_KEY = "Dag ID"
+DAG_AND_TASK_ID_METADATA_KEY = "airlift/dag_and_task_ids"
```

Comment:
In a follow up, let's change the prefix here to be `dagster-airlift`, the name of the publish pypi package, as that provides a convenient way to namespace tags and metadata in an enforceable way.